include/IndexStoreDB/Core/Symbol.h

```diff
@@ -63,6 +63,7 @@ enum class SymbolKind : uint8_t {
   ConversionFunction,
 
   CommentTag,
+  Concept,
```

Comment:
Stylistically I would put this after ConversionFunction, similar to how it's organized in indexstore.h

Reply:
I moved Concept before CommentTag in all the switches/enums.